### DISCUSSION TITLE
Is it intentional that "Metadata" links to mkdocs-material/src/base.html instead of the Meta extension?

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#2193] [Q&A] [answered]
When viewing the Meta Tags page, does the word "Metadata" link to [mkdocs-material/src/base.html](https://github.com/squidfunk/mkdocs-material/blob/master/src/base.html)
Considering this word is used in the sentence "The Metadata extension, which is part of the standard Markdown library, ..." did I expect it to link to the [Meta-Data page](https://python-markdown.github.io/extensions/meta_data/) of the Python Markdown library and not to a page in the mkdocs-material repository.

Is this intentional considering other pages mentioning parts of the Markdown lib link to their respective page (i.e. the attr_list one)?

**Comment by squidfunk** [comment, marked as answer]
No, not intentional. Probably the links were not adjusted. Feel free to create a PR 😊 Other links may also be impacted.
